src/metadataGeneration/extraTagsGenerator.ts

```diff
@@ -0,0 +1,69 @@
+import { Type, Parameter } from './metadataGenerator';
+import * as ts from 'typescript';
+import * as tsserver from '../tsserver';
+

```

Comment:
Looks like an extra space is here; make sure to run the lint task to catch any other problems.